tests/src/test/java/org/eclipse/hono/tests/client/ClientTestBase.java

```diff
@@ -50,15 +49,12 @@
  */
 public abstract class ClientTestBase {
 
-    /**
-     * The identifier of the device used throughout the test cases.
-     */
-    protected static final String DEVICE_ID = "device-0";
+    protected static final String TEST_TENANT_ID = System.getProperty(IntegrationTestSupport.PROPERTY_TENANT, Constants.DEFAULT_TENANT);
+    protected static final Vertx  vertx = Vertx.vertx(new VertxOptions().setBlockedThreadCheckInterval(1000 * 60 * 60));
+    static final long   DEFAULT_TEST_TIMEOUT = 20000; // ms
```

Comment:
is this used somewhere?

Reply:
yes, it use in the test case for that class when calculating the time a thread should wait for all messages to be consumed